pkg/fanal/analyzer/language/java/sbt/lockfile.go

```diff
@@ -0,0 +1,51 @@
+package sbt
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/trivy/pkg/dependency/parser/sbt/lockfile"
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer/language"
+	"github.com/aquasecurity/trivy/pkg/fanal/types"
+)
+
+func init() {
+	analyzer.RegisterAnalyzer(&sbtDependencyLockAnalyzer{})
+}
+
+const (
+	version      = 1
+	requiredFile = "build.sbt.lock"
+)
+
+// sbtDependencyLockAnalyzer analyzes '*.sbt.lock'
+type sbtDependencyLockAnalyzer struct{}
+
+func (a sbtDependencyLockAnalyzer) Analyze(_ context.Context, input analyzer.AnalysisInput) (*analyzer.AnalysisResult, error) {
+	filePath := filepath.Join(input.Dir, input.FilePath)
+	parser := lockfile.NewParser()
+
+	res, err := language.Analyze(types.Sbt, filePath, input.Content, parser)
```

Comment:
`input.FilePath` is enough.

```suggestion
	filePath := filepath.Join(input.Dir, input.FilePath)
	parser := lockfile.NewParser()

	res, err := language.Analyze(types.Sbt, filePath, input.Content, parser)
```

Reply:
Done https://github.com/aquasecurity/trivy/pull/6882/commits/23ce7e672b2d25fbdad9a3bfd4e2c8cb5a2c71f8